core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -28,8 +28,10 @@
     </div>
 
     <div ng-if="rule.type == 'FuzzyMatches'">
-      <classifier-panel rule-inputs="rule.inputs" on-training-data-deletion="onDeleteTrainingDataEntry(index)">
```

Comment:
One more quick check. We've removed classifier-panel and replaced it with some new stuff. Are we losing any functionality by doing this? There's some stuff in the deleted file along the lines of $scope.openRetrainAnswerModal and I wondered if the behavior is regressing.